itemcollection-spec/itemcollection-spec.md

```diff
@@ -27,7 +27,18 @@ This object describes a STAC ItemCollection. The fields `type` and `features` ar
 | type       | string | **REQUIRED.** always "FeatureCollection" to provide compatability with GeoJSON  |
 | features   | [Item] | **REQUIRED** a possibly-empty array of Items          |
 | links      | [Link] | an array of Links related to this ItemCollection   |
-          
+| properties      | Properties | an array of Links related to this ItemCollection   |
+
+### Properties Object
+
+The Properties object adds additional metadata to the ItemCollection. Basically, each entry is a
+key-value pair. The values SHOULD not be an array or object to avoid GIS systems mis-rendering them.
```

Comment:
We are not following this recommendation in Items any longer so I think we can also leave it out here.

Reply:
Okay, we should also remove from Items spec (since I copy/pasted from there)

Reply:
Oh, is it? We can still (should?) have a recommendation that explains why arrays/objects are problematic, but a SHOULD seems contradicting as some of our own specifications and extensions use arrays/objects. (e.g. bands).

Reply:
Maybe a more precise recommendation should be to use a nested data structure only when the data itself is actually nested (as with bands), but not when you just want to group a set of related attributes together.  Related attributes should be flat and to use prefixes on the attributes (as we do with extensions) when you just want to group them.

Reply:
Sounds good for me. That's exactly the ruleset we also just applied to the SAR extension, which used quite a lot of arrays for grouping. See #422.

Reply:
Updated the language